src/kernel.ts

```diff
@@ -547,25 +576,6 @@ class Kernel {
     };
   }
 
-  /**
-   * Close the Websocket.
-   */
-  private _stopChannels(): void {
```

Comment:
the global renaming of `_stopChannels` to `close` hurt my brain as I feel like too many things are called `close` also it move from private to public. I'm fine with this too. just saying.